ophyd/controls/positioner.py

```diff
@@ -640,6 +655,15 @@ def report(self):
         return {self._name: self.position, 'pv': self._readback.pvname}
 
     @property
+    def describe(self):
+        desc = self._readback.describe[self._readback.name]
+        return {self.name: desc}
+
+    def read(self):
```

Comment:
Hmm, I wouldn't expect read to be a property because it is a verb. Maybe @tacaswell suggested this because `describe` is currently a property. I would be in favor of making `describe()` a method and leaving `read()` a property so they are symmetric.